jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/executor/PolicyExecutor.java

```diff
@@ -62,7 +62,8 @@
 import software.amazon.awssdk.services.sns.model.PublishResponse;
 import static com.tmobile.pacman.common.PacmanSdkConstants.JOB_NAME;
 import static com.tmobile.pacman.commons.PacmanSdkConstants.DATA_ALERT_ERROR_STRING;
-
+import static com.tmobile.pacman.commons.PacmanSdkConstants.ENDING_QUOTES;
+import static com.tmobile.pacman.commons.PacmanSdkConstants.ERROR_MESSAGE;

```

Comment:
What is need for the \" surrounding the error message?

Reply:
datadog needs error messages inside quotes